app/views/parks/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if params[:search].present? %>
```

Comment:
The thought process on this is that I don't want to display the page unless there is a search conditional. So once the user selects a city the page will be displayed. 

If they decide "Hmm I'm thinking about moving to this other city or I'm going to be visiting this city" then they can edit the search field on the results page to show parks in the other city.